server/src/services/cronSchedule/cronSchedule.js

```diff
@@ -8,6 +8,38 @@ import { Applicant } from '../../models/Applicant.js';
 import { Transaction } from '../../models/Transaction.js';
 import { createTransaction, eventHandling } from '../checkResult/checkResult.js';
 
+const fetchTransaction = async (txId) => {
+  console.log(`Transaction ID ${txId}`);
+  const response = await axios.get(`${EXPERIENCE_TRANSACTION_URL}/${txId}`, {
+    headers: {
+      'x-trulioo-api-key': truliooApiKey,
+    },
+  });
+
+  const status = response.data ? response.data.status : 'unknown';
+  console.log(`Status: ${status}`);
+
+  return { status, steps: response?.data?.steps };
+};
+
+/**
+ * // TODO: Document what this is doing and why.
+ *
+ * @param {Array} steps
+ * @returns {object}
+ */
+// eslint-disable-next-line unicorn/prevent-abbreviations
+const getEmbedIDDocVStep = (steps) => {
+  return steps.find((element) => {
+    return element.transactionType === 'EmbedID' && element.stepName === 'DocVStep';
+  });
+};
+
+/**
+ * // TODO: Document what this is doing and why.
+ *
```

Comment:
Processing of applicants for whom a document check has been initiated to find data on completed trulioo transactions.

Reply:
@fedencer Can you please explain this one further? Maybe break down the pieces of the function and explain what the different sections do?

Thanks!

@sandoche To me, a JSDoc of `Update status of document verification` sounds like it's describing a simple setter function rather than something complicated like we see here.